src/main/java/org/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java

```diff
@@ -73,7 +73,7 @@ public CascadePermission getPermission() {
 
     @Override
     public String description() {
-        return "Soft-ban a user";
+        return "Bans and then unbans a specified user to remove messages";
```

Comment:
`to remove messages and kick`

Reply:
If we're going to add kick it needs to be `in order to remove messages and kick the user`